### PR TITLE
Add the Zurich chapter

### PR DESCRIPTION
**Pull request by lehmannro**
Not much to see here but we better have some central landing point.

### CODE REVIEW

File: zurich/index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: city
+title: Zürich
+class: "zurich"
+tagline: "The real Mountain View"
```
Comment (gnunicorn) [inline zurich/index.md:5]: LOVE IT!

File: zurich/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+rss_feed: zurich
```
Comment (gnunicorn) [inline zurich/index.md:6]: need to add this to the OTS-Blog then, too

**Comment by gnunicorn** [comment]
I think we are still missing the compiles css file for this to work, right? Will you update this or should I?

File: zurich/index.md (hunk 2 above)
Comment (gnunicorn) [inline zurich/index.md:6]: done. see https://github.com/OpenTechSchool/blog.opentechschool.org/commit/cd31abbd53b175cc24a4d8efa56549e6a247442a

File: _includes/_footer.html [hunk 3]
```diff
@@ -17,6 +17,9 @@
             <li>
               <a href="/hamburg/">Hamburg</a>
             </li>
+            <li>
+              <a href="/zurich/">Zürich</a>
```
Comment (gnunicorn) [inline _includes/_footer.html:21]: Was actually just thinking... we didn't have that case yet... As for city naming do we want to use english (as the entire website is anyways) or use the mother tongue - which then rases the questions of whether to use Zürich or Züri ;) . Not a blocker, was just thinking about this ...

**Comment by lehmannro** [comment]
Could you regenerate the CSS files?

File: _includes/_footer.html (hunk 3 above)
Comment (lehmannro) [inline _includes/_footer.html:21]: Good point!  I think I'd chose "zurich" as an identifier (say, in URIs) and "Zürich" as a label.  No strong feelings here though and I think most people will just happily ignore the umlauts. :-)

**Comment by gnunicorn** [comment]
superseeded by #109

**Pull request closed without merging**